build/maven.go

```diff
@@ -211,6 +218,17 @@ func (mm *MavenModule) loadMavenHome() (mavenHome string, err error) {
 	return
 }
 
+func (mm *MavenModule) getSuitableM2String() string {
+	maven := "mvn"
+	if mm.extractorDetails.useWrapper {
+		maven, _ = filepath.Abs("./mvnw")
+		if runtime.GOOS == "windows" {
+			maven, _ = filepath.Abs("mvnw.cmd")
+		}
```

Comment:
The following code is more efficient, becuase it avoids two `filepath.Abs` in the case of Windoes - 
```
		if runtime.GOOS == "windows" {
			maven, _ = filepath.Abs("mvnw.cmd")
		} else {
			maven, _ = filepath.Abs("./mvnw")
		}
```